tools/test/travis-ci/scancode_evaluate_test.py

```diff
@@ -34,22 +34,36 @@
  * limitations under the License.\
  */"
 
+BINARY_HEADER = "/*\
```

Comment:
Just to be in line with the other header constants.
```suggestion
HEADER_WITH_BINARY_LICENSE = "/*\
```